src/indicators/drought/chirps_rainfallobservations.py

```diff
@@ -214,63 +239,94 @@ def compute_seasonal_lowertercile_raster(
         country_iso3, config
     )
     (
-        chirps_seasonal_lowertercile_country_filepath,
-        chirps_seaonal_tercile_bounds_country_filepath,
-    ) = get_filepath_seasonal_lowertercile_raster(country_iso3, config)
+        chirps_seasonal_tercile_filepath,
+        chirps_seasonal_tercile_bounds_filepath,
+    ) = get_filepath_seasonal_tercile_raster(country_iso3, config)
 
-    if use_cache and chirps_seasonal_lowertercile_country_filepath.exists():
+    if use_cache and chirps_seasonal_tercile_filepath.exists():
         logger.debug(
-            f"{chirps_seasonal_lowertercile_country_filepath} already exists"
+            f"{chirps_seasonal_tercile_filepath} already exists"
             " and cache is set to True, skipping"
         )
-        return chirps_seasonal_lowertercile_country_filepath
+        return chirps_seasonal_tercile_filepath
 
-    Path(chirps_seasonal_lowertercile_country_filepath.parent).mkdir(
+    Path(chirps_seasonal_tercile_filepath.parent).mkdir(
         parents=True, exist_ok=True
     )
     logger.debug("Computing lower tercile values...")
+    # for some unknown reason the rolling sum takes very long to
+    # compute when using rioxarray so sticking to xarray for now
     ds = xr.load_dataset(chirps_monthly_country_filepath)
+    da = ds.precip
```

Comment:
You can just tack this on to the end of the previous line and avoid defining `ds`